packit/api.py

```diff
@@ -2031,6 +2031,50 @@ def watch_copr_build(
             report_func=report_func,
         )
 
+    def run_osh_build(
+        self,
+        chroot: Optional[str] = "fedora-rawhide-x86_64",
+        srpm_path: Optional[Path] = None,
+        upstream_ref: Optional[str] = None,
+        release_suffix: Optional[str] = None,
+        srpm_version_diff: Optional[str] = None,
+        comment: Optional[str] = None,
+    ) -> str:
+        """
+        """
+        import subprocess
+
+        if not srpm_path:
+            srpm_path = self.create_srpm(
+                upstream_ref=upstream_ref,
+                srpm_dir=self.up.local_project.working_dir,
+                release_suffix=release_suffix,
+            )
+
```

Comment:
if this will be used from the service part later, I think we should probably initialise the Kerberos ticket with [this method](https://github.com/packit/packit/blob/ce6bd5775ae5deb99260bfcee240febf49151b4e/packit/api.py#L2092)